### ISSUE TITLE
Use SAM-Med3D encoder for inference without prompts for automatic segmentation

### ISSUE THREAD

**Issue by SergioRodLla** [uni-medical/SAM-Med3D#79]
Hi, thanks for all the work done!

Is it possible to leverage just the SAM-Med3D encoder (with sam_med3d_turbo) for inference without click prompts? I want to use its encoder with a custom decoder in an automatic segmentation task. My idea is to fine-tune a decoder only, while taking advantage of the features extracted by SAM-Med3D encoder (frozen). Would click prompts still be needed in this case?

Best,
Sergio

**Comment by blueyo0** [comment]
Hi, good idea! SAM-Med3D has the potential to work as a powerful pre-training for 3D medical automatic segmentation. You can fine-tune the decoder only and discard the prompt encoder. 

Here's some reference results for transfer learning:
![image](https://github.com/user-attachments/assets/4525028e-d125-42a9-9f95-612ff5289ca1)

**Comment by SergioRodLla** [comment]
Hi. Thanks for the feedback!

**Comment by blueyo0** [comment]
😄 Ok, close this issue now since it is solved.
